### DISCUSSION TITLE
只键盘本体的话，是否有改成三模的可能性？

### DISCUSSION THREAD

**Discussion by vvrfxyz** [peng-zhihui/HelloWord-Keyboard#128] [Q&A]
因为对扩展性没啥要求，只是对这个电路设计感兴趣，同时想改三模连接，但因为对电路设计只有大学一点点基础水平，所以想看有没有大佬解答一下

**Comment by Shao-YiLin** [comment]
这个键盘由于使用移位寄存器扫描按键，无线连接进入低功耗模式，无法用普通按键唤醒，不考虑拓展模块只可以使用FN键（直连键盘）唤醒。如果想用全键唤醒键盘的话就老老实实用矩阵扫描的方式吧。

**Comment by alexhao007** [comment]
我感觉更要关注一下功耗吧
